Formula/kustomize.rb

```diff
@@ -0,0 +1,31 @@
+class Kustomize < Formula
+  desc "Template-free customization of Kubernetes YAML manifests"
+  homepage "https://github.com/kubernetes-sigs/kustomize"
+  url "https://github.com/kubernetes-sigs/kustomize.git",
+      :tag => "v1.0.6",
+      :revision => "017c4ae0aa19195db2a51ecc5aa82c56a1f1c99b"
+  head "https://github.com/kubernetes-sigs/kustomize.git"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["CGO_ENABLED"] = "0"
+
+    revision = `git rev-parse HEAD`.strip
+    tag = `git describe --tags`.strip
+    dir = buildpath/"src/github.com/kubernetes-sigs/kustomize"
+    dir.install buildpath.children - [buildpath/".brew_home"]
+    cd dir do
+      system "go", "install", "-ldflags",
+             "-s -X github.com/kubernetes-sigs/kustomize/pkg/commands.kustomizeVersion=#{tag}"\
```

Comment:
Use [this kind of style](https://github.com/Homebrew/homebrew-core/blob/de2d8e85c4ec432c3b9742bb3a970dc9ea0da13a/Formula/kubernetes-service-catalog-client.rb#L25-L29) please.

Reply:
Done.